pandas/tests/reshape/test_pivot.py

```diff
@@ -1656,6 +1656,49 @@ def test_categorical_margins_category(self, observed):
         table = df.pivot_table("x", "y", "z", dropna=observed, margins=True)
         tm.assert_frame_equal(table, expected)
 
+    def test_pivot_with_categorical(self, observed, ordered_fixture):
+        # gh-21370
+        idx = [np.nan, "low", "high", "low", np.nan]
```

Comment:
you are not using observed here?

Reply:
@jreback Do you mean pass as keyword arg to `pivot_table()` ? Ive just pushed changes